src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+        System.out.println("Hello I'm Duke\nWhat can I do for you?");
+        Scanner scan = new Scanner(System.in);
+        
+        while (true) {
+            String textInput = scan.nextLine();
+
+            if (textInput.equalsIgnoreCase("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                scan.close();
+                return;
+            }
+
+            if (textInput.equalsIgnoreCase("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskList.size(); i++) {
+                    System.out.println(i + 1 + ". " + taskList.get(i).toString());
+                }
+                continue;
+            }
+
+            if (textInput.length() >= 8 &&
+                    textInput.substring(0, 6).equalsIgnoreCase("delete")) {
+                int i = Integer.parseInt(textInput.substring(7));
+                Task t = taskList.get(i - 1);
+                taskList.remove(i - 1);
+                String output = String.format("Got it. I've removed this task:\n%s\nNow you have %d tasks in the list", t.toString(), taskList.size());
+                System.out.println(output);
+                continue;
```

Comment:
I find the use of multiple `continue` instructions in the code strange. The while loop already has a looping behaviour once it reaches the end of the loop, so using `continue` to return to the start of the loop should not be a default behaviour. What is your rationale for designing it this way?

Rather, I would suggest the following structure:
```java
boolean isRunning = true;
while (isRunning) {
    if (textInput.equalsIgnoreCase("bye")) {
        isRunning = false;
    } else if (stmt) {
        code;
    } else {
        code;
    }
}
```

Alternatively, you may consider parsing the command first, then using a switch-case statement to handle various cases.